frontend/src/App.tsx

```diff
@@ -15,7 +15,7 @@ function App (): JSX.Element {
         <Route path="/login" element={<LoginForm />} />
         <Route path="/signup" element={<SignUp />} />
         <Route path="/tempLandingPage" element={<TempLandingPage />} />
-        <Route path="/contactPage" element={<Contacts />} />
+        <Route path="/contactPage" element={<ContactsList />} />
```

Comment:
nit: the url doesn't seem right to me if it's called `wewrap.com/contactPage` I think it'd normally just be `wewrap.com/contacts`

Reply:
Yeah I agree will change!